operations/mimir/config.libsonnet

```diff
@@ -367,6 +388,8 @@
 
     // Enables query-scheduler component, and reconfigures querier and query-frontend to use it.
     query_scheduler_enabled: true,
+    query_scheduler_service_discovery_mode: 'dns',
```

Comment:
Let's document available options here?

Reply:
Definitely, good idea.

Reply:
Done in https://github.com/grafana/mimir/pull/3128/commits/b3b21959d78d6a0e8fbd11d40ce1ebe1b7fd6647